tests/tests_server.py

```diff
@@ -435,6 +436,31 @@ def get_ns_of_nodes(nodes):
 
         self.assertRaises(ValueError, ua_utils.get_nodes_of_namespace, self.opc, namespaces='non_existing_ns')
 
+    def test_silent_events(self):
+        myhandler = MySubHandler()
+        o = self.opc.get_objects_node()
+
+        v1 = o.add_variable(3, 'SubscriptionVariableSE', 0)
+        sub = self.opc.create_subscription(100, myhandler)
+
+        handle1 = sub.subscribe_data_change(v1)
+
+        myhandler.reset()
+        v1.set_value(5)
+        node, val, data = myhandler.future.result(timeout=10)
+        self.assertEqual(val, 5)
+
+        myhandler.reset()
+
+        # TODO: this feels not good. Maybe added set_silent_events to a higher class ?
+        self.opc.iserver.isession.set_silent_events(True)
```

Comment:
Add a helper method to server. But I am wondering about the name. Maybe disable_data_change_notification(true) is more explicit